src/Propel/Generator/Builder/Om/templates/baseObjectMethods.php

```diff
@@ -171,6 +171,8 @@ public function setVirtualColumn($name, $value)
      * @param  string  $msg
      * @param  int     $priority One of the Propel::LOG_* logging levels
      * @return boolean
+     *
+     * @deprecated
```

Comment:
It's good practice to write version in which method will be deleted and message like `Consult http://propelorm.org/documentation/07-logging.html`